lib/tasks/cap_cutover.rake

```diff
@@ -1,79 +1,10 @@
 require 'csv'
 require 'dotiw'
-require 'activerecord-import'
 
-namespace :cap_cutover do
-  # to reubuild:
-  # run pull_sw_for_cap, pull_pubmed_for_cap, ingest_authors, create_pubs_from_cap, build_pubs, ingest_man_pubs
-  # then run cap_authorship_
-
-  desc 'get all pubmed source records for full cap dump'
-  task :pull_pubmed_for_cap, [:file_location] => :environment do |_t, args|
-    include ActionView::Helpers::DateHelper
-    start_time = Time.zone.now
-    total_running_count = 0
-    new_downloads = 0
-    pmids_for_this_batch = Set.new
-    cap_import_pmid_logger = Logger.new(Rails.root.join('log', 'cap_import_pubmed_source_records.log'))
-    cap_import_pmid_logger.info 'Started pumed import ' + Time.zone.now.to_s
-    lines = CSV.foreach(args.file_location, headers: true, header_converters: :symbol) do |row|
-      total_running_count += 1
-      pmid = row[:pubmed_id].to_s
-      pmids_for_this_batch << pmid unless PubmedSourceRecord.exists?(pmid: pmid)
-      if pmids_for_this_batch.length % 4000 == 0
-        PubmedSourceRecord.get_and_store_records_from_pubmed(pmids_for_this_batch)
-        new_downloads += pmids_for_this_batch.size
-        pmids_for_this_batch.clear
-        Rails.logger.debug "#{total_running_count} in #{distance_of_time_in_words_to_now(start_time)}"
-        # break
-      end
-      GC.start if total_running_count % 4000 == 0
-    end
-    # finish off the batch
-    PubmedSourceRecord.get_and_store_records_from_pubmed(pmids_for_this_batch)
-    new_downloads += pmids_for_this_batch.size
-    Rails.logger.info total_running_count.to_s + 'records were processed in ' + distance_of_time_in_words_to_now(start_time, true)
-    Rails.logger.info lines.to_s + ' lines of file: ' + args.file_location.to_s + ' were processed.'
-    Rails.logger.info new_downloads.to_s + 'new records downloaded'
-    cap_import_pmid_logger.info 'Finished pubmed import.' + Time.zone.now.to_s
-    cap_import_pmid_logger.info lines.to_s + ' lines of file: ' + args.file_location.to_s + ' were processed.'
-    cap_import_pmid_logger.info total_running_count.to_s + 'records were processed in ' + distance_of_time_in_words_to_now(start_time, true)
-    cap_import_pmid_logger.info new_downloads.to_s + ' new records downloaded'
-  end
```

Comment:
OK, `pull_pubmed_for_cap` noted for deprecation (removal) in https://github.com/sul-dlss/sul_pub/issues/356#issuecomment-341251365